salt/base/config/publish-files-nginx.conf

```diff
@@ -0,0 +1,13 @@
+server {
+  listen 80 ssl default_server;
+
+  ssl_certificate /etc/ssl/private/salt.psf.io.pem;
+  ssl_certificate_key /etc/ssl/private/salt.psf.io.pem;
+
+  server_name _;
```

Comment:
Let's choose a hostname, which will allow us to remove `default_server` above and set `server_name` to something precise.

Let's go with `salt.psf.io`.

Reply:
perhaps `salt-public.psf.io` to keep from confusing the internal name we already use (salt.psf.io).